test/e2e/README.md

```diff
@@ -30,26 +30,24 @@ knative-gcp should be added under [knative-gcp e2e test lib](lib).
 
 There are two ways to set up authentication mechanism.
 
-- If you want to run E2E tests with authentication mechanism using
-  **[Workload Identity](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity)**,
-  please follow below instructions to configure the authentication mechanism
-  with **[Workload
-  Identity]**(https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity).
-  **[Workload
-  Identity]**(https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity)
-  is GKE specific.
-- If you want to run E2E tests with authentication mechanism using **Kubernetes
-  Secrets**, please follow below instructions to configure the authentication
-  mechanism with **Kubernetes Secrets**.
+- (GKE Specific) If you want to run E2E tests with **[Workload Identity](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity)**
+  as the authentication mechanism, please follow below instructions to configure the authentication mechanism
+  with **[Workload Identity](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity)**.
+- If you want to run E2E tests with **Kubernetes Secrets** as the authentication mechanism, please
+  follow below instructions to configure the authentication mechanism with **Kubernetes Secrets**.
 
 1.  A running Kubernetes cluster with
     [knative-gcp](../../docs/install/install-knative-gcp.md) installed and
     configured.
-1.  [Pub/Sub Enabled Service Account](../../docs/install/pubsub-service-account.md)
-    installed.
-1.  [GCP Broker Deployment](../../docs/install/install-gcp-broker.md#deployment)
-    and
-    [GCP Broker Authentication Setup](../../docs/install/install-gcp-broker.md#authentication-setup-for-gcp-broker).
+1.  Create a [Pub/Sub Enabled Service Account](../../docs/install/pubsub-service-account.md).
+    Download a credential file and set `GOOGLE_APPLICATION_CREDENTIALS` env var. This is used by
+    some tests(e.g., `TestSmokePullSubscription`) to authorize the Google SDK clients.
+    ```
+    gcloud iam service-accounts keys create cre-pubsub.json \
```

Comment:
Do we have to set the `GOOGLE_APPLICATION_CREDENTIALS` after we create a Pun/Sub Enabled Service Account?

Reply:
Yes, `create a Pun/Sub Enabled Service Account` only asks you to create the GSA, but not use it in local test (by setting the env var)

Reply:
I never set `GOOGLE_APPLICATION_CREDENTIALS`.
I just create the Pub/Sub Enabled Service Account but it works for me.

Reply:
Can you check you `GOOGLE_APPLICATION_CREDENTIALS ` and run `go test --tags=e2e ./test/e2e/... -run TestSmokePullSubscription -v`

Reply:
When I ran `echo $GOOGLE_APPLICATION_CREDENTIAL`, it shows empty. But when I ran `go test --tags=e2e ./test/e2e/... -run TestSmokePullSubscription -v`, it passes.

Reply:
After I fetched the latest code of master branch, I need to provide  `GOOGLE_APPLICATION_CREDENTIALS`, otherwise it will fail

Reply:
Updated to absolute path